RELEASE-NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Wikibase DataModel release notes
 
+## Version 5.0.3 (alpha)
```

Comment:
If this is substantial or not depends a lot. I don't even know if we will see this having any impact. I hope, but I do not know. Do we even have a graf for memory usage?

0.Y steps are for added functionality. This is no added functionality. This is a bugfix, fixing a performance issue.